Firmware/HomeButtonsArduino/sdkconfig.defaults

```diff
@@ -18,3 +18,12 @@ CONFIG_BOOTLOADER_SKIP_VALIDATE_IN_DEEP_SLEEP=y
 CONFIG_BOOTLOADER_APP_ROLLBACK_ENABLE=y
 
 CONFIG_LWIP_DHCP_DOES_ARP_CHECK=n
+
+CONFIG_SPIFFS_OBJ_NAME_LEN=56
+
+CONFIG_MBEDTLS_DYNAMIC_BUFFER=y
+CONFIG_MBEDTLS_DYNAMIC_FREE_PEER_CERT=y
+CONFIG_MBEDTLS_DYNAMIC_FREE_CONFIG_DATA=y
+CONFIG_MBEDTLS_DYNAMIC_FREE_CA_CERT=y
```

Comment:
Open question: is there any alternative without dynamic allocation ?
I haven't checked online yet but maybe you already did.

Reply:
MBEDTLS is allocating a lot of memory. Without setting this config options the min free heap drops to zero basically. 

It's not the best solution but I don't know what else could be done...